src/api/trackerTrackedEntities.ts

```diff
@@ -131,13 +131,16 @@ export type TrackedEntitiesParamsBase = {
     includeDeleted: boolean;
     includeAllAttributes: boolean;
     potentialDuplicate: boolean;
+    order: CommaDelimitedListOfAttributeFilter;
```

Comment:
```
Sortable fields: created (default), lastUpdated, code, uid, id, name, displayName, dimensionType

Direction can be ASC (default) or DESC
```

As the docs are prettify specific, if you have some time for this, it would be nice to use a more narrow type. Allowing this usage would be nice: `{ order: [{field: "code", direction: "asc"}, {field: "name", direction: "desc"}]}`. (and the fields are also checked). What do you think?